tests/worksheet_test.py

```diff
@@ -914,6 +916,130 @@ def test_get_all_records_numericise_unformatted(self):
         d0 = dict(zip(rows[0], expected_values))
         self.assertEqual(read_records[0], d0)
 
+    @pytest.mark.vcr()
+    def test_get_records(self):
+        self.sheet.resize(5, 3)
+        rows = [
+            ["A1", "B1", "C1"],
+            [1, 2, 3],
+            [4, 5, 6],
+            [7, 8, 9],
+            [10, 11, 12],
+        ]
+        cell_list = self.sheet.range("A1:C5")
+        for cell, value in zip(cell_list, itertools.chain(*rows)):
+            cell.value = value
+        self.sheet.update_cells(cell_list)
+
+        # test1 - set max_row only
```

Comment:
these comments are out of date? they are helpful, albeit likely unneeded

Reply:
fixed them, they're up to date now